### ISSUE TITLE
Sprint Goal: Mar 9 - Mar 22

### ISSUE THREAD

**Issue by jmann2817** [CDCgov/data-exchange-upload#74]
Goal  - **Waiting on decision from Ryan on development language** 
- [ ] EDAV to NDLP #54
- [x] Working group session with Keith from IZGW team to discuss suggested modifications and findings. 
- [x] Figure out why IZGW data is not uploaded to edav. 
- [ ] Test tus hook for receiving post-terminate events (upload terminated) #76
- [x] Continue supplemental API work. #68
- [x] convert C# to Kotlin #68 
- [x] Correct issues and test upload function app metadata relay #75 

Stretch goals
- [x] Java client #66
